### PR TITLE
updated to the latest MutationMapper release v1.1.0

### PR DESCRIPTION
**Pull request by onursumer**
# What? Why?
This is a fix for https://github.com/cBioPortal/mutation-mapper/issues/37, https://github.com/cBioPortal/mutation-mapper/issues/43, and https://github.com/cBioPortal/mutation-mapper/issues/44

Changes proposed in this pull request:
- Updates the MutationMapper library to the latest release: https://github.com/cBioPortal/mutation-mapper/releases/tag/1.1.0

# Checks
- [ ] Runs on Heroku.
- [ ] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
If your pull request involves changes to an existing file, one or more of the
people that edited before you might be good candidates to review it. Please use
`git blame <filename>` to determine that and notify them here. Otherwise notify
everybody in the core team:
@cBioPortal/core-developers

### CODE REVIEW

**Comment by pieterlukasse** [comment]
@jjgao can we merge this one soon? I scanned the 508 related changes and these seem OK.  The other changes seem fine as well (like minor refactoring/fixes and color scheme changes).
I tested the mutations view table header qtips fix and it works fine now, except for the "Annotation" column which shows no qtip. @onursumer maybe you can fix this in a new PR? Then we could merge this one, which is already an improvement. :+1:

File: portal/src/main/webapp/js/lib/mutationMapper.js [hunk 1]
```diff
@@ -1573,18 +1574,26 @@ var PileupUtil = (function()
...
+				count: groupCountMap[group],
+				priority: mutationTypeMap[group].priority});
 		});
 
 		groupArray.sort(function(a, b) {
-			// descending sort
-			return b.count - a.count;
+			if (b.count === a.count) {
+				// tie condition: use mutation type priority
+				return b.priority - a.priority;
```
Comment (pieterlukasse) [inline portal/src/main/webapp/js/lib/mutationMapper.js]: perhaps double check: is `.priority` *always* set?

**Comment by onursumer** [comment]
@pieterlukasse Annotation column was actually _plugged in_ by the portal code base, it is not a part of the MutationMapper library. The initial decision was to have no tool tip for this column.

@zhx828 can you suggest a basic description for the annotation column tool tip?

**Comment by zhx828** [comment]
@pieterlukasse We intended to do that without tooltip in the annotation column. But that was back when we only have very few annotation sources. @jjgao maybe it's time to have a tooltip now? We may have mycancergenome, hotspot, OncoKB, MolecularMatch and Civic in the future, and all will be squeezed into annotation column.

File: portal/src/main/webapp/js/lib/mutationMapper.js (hunk 1 above)
Comment (onursumer) [inline portal/src/main/webapp/js/lib/mutationMapper.js]: We are setting the priority a few lines above: https://github.com/cBioPortal/cbioportal/pull/1325/files#diff-e64faf74f1b5d548c7cb04ee4bd3aaa2R1585

And the source of this priority is a static map: https://github.com/cBioPortal/mutation-mapper/blob/master/src/js/util/MutationViewsUtil.js#L43

Currently, I don't see a case where the priority can be undefined. But it is always possible that sometime in the future a new entry without a priority value into this static map can break the code :)

File: portal/src/main/webapp/js/lib/mutationMapper.js [hunk 2]
```diff
@@ -1559,12 +1560,12 @@ var PileupUtil = (function()
```
Comment (pieterlukasse) [inline portal/src/main/webapp/js/lib/mutationMapper.js]: you forgot to update the comment 2 lines above

File: portal/src/main/webapp/js/lib/mutationMapper.js (hunk 1 above)
Comment (pieterlukasse) [inline portal/src/main/webapp/js/lib/mutationMapper.js]: Ok, seems correct :+1:

File: portal/src/main/webapp/js/lib/mutationMapper.js (hunk 2 above)
Comment (onursumer) [inline portal/src/main/webapp/js/lib/mutationMapper.js]: good catch!

**Pull request merged**